src/libxrpl/protocol/TxFormats.cpp

```diff
@@ -374,6 +374,15 @@ TxFormats::TxFormats()
         },
         commonFields);
 
+    add(jss::NFTokenModify,
+        ttNFTOKEN_MODIFY,
+        {
+            {sfNFTokenID, soeREQUIRED},
+            {sfOwner, soeOPTIONAL},
+            {sfURI, soeOPTIONAL},
```

Comment:
Since you pointed it out, as of Jun 29 the `sfURI` field has been updated to an optional field.

Reply:
Is there any use case for removing the URI from an existing NFT? Is there any difference from burning the NFT after it becomes an NFT without an URI?

Reply:
Such a use case may not be common, but it is not impossible to say that there is no use case.

Since the NFTokenMint transaction can be executed with the URI field unspecified, it is reasonable that the NFTokenModify transaction can change the URI field to unspecified.

Reply:
Should there be a flag for removing the URI from the NFT, like with `NFTokenMinter`, instead of just using its absence?